pkgs/applications/editors/emacs/elisp-packages/melpa-packages.nix

```diff
@@ -174,6 +174,10 @@ let
         # upstream issue: missing package version
         cmake-mode = dontConfigure super.cmake-mode;
 
+        consult-gh = super.consult-gh.overrideAttrs(old: {
+          propagatedUserEnvPkgs = old.propagatedUserEnvPkgs or [ ] ++ [ pkgs.gh ];
```

Comment:
not sure if `meta.maintainers` should be kept  cc @AndersonTorres

Reply:
Well, since it will be melpified and therefore automated, I relinquish the maintenance :)